README.md

```diff
@@ -1,3 +1,5 @@
+[![Silabs upstream daily build](https://github.com/SiliconLabsSoftware/zephyr-silabs/actions/workflows/upstream-build.yml/badge.svg)](https://github.com/SiliconLabsSoftware/zephyr-silabs/actions/workflows/upstream-build.yml)
```

Comment:
This is a rather long line. I believe you can get the same result with:

```
[![Silabs upstream daily build][badge]][recipe]

[badge]:  https://github.com/SiliconLabsSoftware/zephyr-silabs/actions/workflows/upstream-build.yml/badge.svg
[recipe]: https://github.com/SiliconLabsSoftware/zephyr-silabs/actions/workflows/upstream-build.yml
```

Reply:
@jerome-pouiller sure, thanks. What I initially had was a copy-paste of what GitHub itself offers when you select "Generate badge" in the Actions tab.

Reply:
@jerome-pouiller done now, however I trust you that it's actually valid syntax (I'm nota Markdown expert)

Reply:
Looking at https://github.com/jhedberg/zephyr-silabs/tree/upstream_status_badge it seems to be fine now.

Reply:
If someone fork `zephyr-silab` he will get the status of the upstream repo. It is not a big deal I assume.